src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-7-0-319.mdx

```diff
@@ -9,6 +9,15 @@ security: []
 ---
 ## New Relic PHP Agent v10.7.0.319
 
+### Important note
+
+If the following error occurs when installing the newrelic-php5 Debian package:
+
+    Setting up newrelic-php5 (10.7.0.319) ...
```

Comment:
@mfulb What's the (...) supposed to indicate?

Reply:
@mfulb Would it be appropriate to put it in code format then? Or just leave it as is?

Reply:
I would like those two lines to be in a preformatted "code" block so it looks like screen output you would see running the installation program.

Reply:
I'll doublecheck how the preview works to make sure it's formatted for code. Just to confirm, should the syntax highlighting be in C?

Reply:
It is output on a terminal screen I don't think it needs any highlighting.  Just fixed spaced preformatted text.